CTP/shell/src/com/navercorp/cubridqa/shell/main/Test.java

```diff
@@ -284,13 +284,12 @@ public String runTestCase_linux() throws Exception {
 
 		script = new ShellInput("cd " + testCaseDir);
 		script.addCommand("ulimit -c unlimited");
-		script.addCommand("export JAVA_HOME=$JAVA_HOME_" + (context.getVersion().trim().toUpperCase()));
+		script.addCommand("if [ \"$JAVA_HOME_" + context.getVersion().trim().toUpperCase() + "\" ]; then");
+		script.addCommand("        export JAVA_HOME=$JAVA_HOME_" + (context.getVersion().trim().toUpperCase()));
+		script.addCommand("fi");
 		script.addCommand("export PATH=$JAVA_HOME/bin:$PATH");
-		script.addCommand("export QA_REPOSITORY=$HOME/CQT");
-		script.addCommand("export init_path=$QA_REPOSITORY/lib/shell/common");
-		script.addCommand("export SHELL_CONFIG_PATH=$QA_REPOSITORY/lib/shell/common");
-		script.addCommand("export CLASSPATH=$HOME/CUBRID/jdbc/cubrid_jdbc.jar:$HOME/CQT/lib/shell/common/commonforjdbc.jar:.");
-		script.addCommand("export LD_LIBRARY_PATH=$QA_REPOSITORY/lib/shell/common/commonforc/lib:$LD_LIBRARY_PATH");
+		script.addCommand("export CLASSPATH=$CUBRID/jdbc/cubrid_jdbc.jar:$init_path/commonforjdbc.jar:.");
```

Comment:
As is:
+		script.addCommand("export CLASSPATH=$CUBRID/jdbc/cubrid_jdbc.jar:$init_path/commonforjdbc.jar:."); +		script.addCommand("export LD_LIBRARY_PATH=$init_path/commonforc/lib:$LD_LIBRARY_PATH");

To be:
It's better to move to init function in init.sh